### DISCUSSION TITLE
How to run setup

### DISCUSSION THREAD

**Discussion by esphung** [kiraio-moe/NikkeViewerEX#1] [Q&A]

**Comment by kiraio-moe** [comment]
Well, first off you need to build the project. The project is built on top of Unity game engine.
Then you need to find the raw game assets. You can find them at [@nikke-db](https://github.com/Nikke-db/Nikke-db.github.io/tree/main/l2d) if you want. The rest just follow the instruction at the README (hope I described it clearly :) )
